.gitignore

```diff
@@ -5,5 +5,5 @@
 /releases
 deployments/helm/gpu-feature-discovery
 e2e_logs
-
 *.out
+*.tgz
```

Comment:
I removed this because this causes the nfd helm `tgz` to be ignored when updating dependencies. Is there a specific reason to add this?